### PR TITLE
Fix spelling error

### PR DESCRIPTION
**Pull request by duytrinhvn**
- [ x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [ x] None of my changes are plagiarized from another source without proper attribution.
- [x ] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [ x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-and-save-a-record-of-a-model.english.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ In this challenge you will have to create and save a record of a model.
 
 ## Instructions
 <section id='instructions'>
-Create a document instance using the <code>Person</code> constructor you built before. Pass to the constructor an object having the fields <code>name</code>, <code>age</code>, and <code>favoriteFoods</code>. Their types must be conformant to the ones in the Person Schema. Then call the method <code>document.save()</code> on the returned document instance. Pass to it a callback using the Node convention. This is a common pattern, all the following CRUD methods take a callback function like this as the last argument.
```
Comment (RandellDawson) [inline curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-and-save-a-record-of-a-model.english.md]: Conformant is an adjective which seems OK in this sentence.  Conformed seems odd to me.  How about the following?
```suggestion
Create a document instance using the <code>Person</code> constructor you built before. Pass to the constructor an object having the fields <code>name</code>, <code>age</code>, and <code>favoriteFoods</code>. Their types must conform to the ones in the Person Schema. Then call the method <code>document.save()</code> on the returned document instance. Pass to it a callback using the Node convention. This is a common pattern, all the following CRUD methods take a callback function like this as the last argument.
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
See my comment.

File: curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-and-save-a-record-of-a-model.english.md (hunk 1 above)
Comment (duytrinhvn) [reply curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-and-save-a-record-of-a-model.english.md]: Yes, "Their types must conform" is easier to understand, especially for the second language people like me.

Comment (duytrinhvn) [reply curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-and-save-a-record-of-a-model.english.md]: Yes, "Their types must conform" is easier to understand, especially for people who English is the second language like me.

**Review by thecodingaviator — APPROVED** [review]
LGTM

**Comment by RandellDawson** [comment]
@thecodingaviator Are you approving "Must be conformed" (what the OP has now) or what I suggested "Must conform" (which has not been committed yet?

**Comment by duytrinhvn** [comment]
"Must conform" is good.

**Comment by thecodingaviator** [comment]
@RandellDawson My bad, I should've included that I'm approving it with the changes you've requested

File: curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-and-save-a-record-of-a-model.english.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ In this challenge you will have to create and save a record of a model.
...
+Create a document instance using the <code>Person</code> constructor you built before. Pass to the constructor an object having the fields <code>name</code>, <code>age</code>, and <code>favoriteFoods</code>. Their types must be conformed to the ones in the Person Schema. Then call the method <code>document.save()</code> on the returned document instance. Pass to it a callback using the Node convention. This is a common pattern, all the following CRUD methods take a callback function like this as the last argument.
```
Comment (thecodingaviator) [inline curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-and-save-a-record-of-a-model.english.md]:
```suggestion
Create a document instance using the <code>Person</code> constructor you built before. Pass to the constructor an object having the fields <code>name</code>, <code>age</code>, and <code>favoriteFoods</code>. Their types must conform to the ones in the Person Schema. Then call the method <code>document.save()</code> on the returned document instance. Pass to it a callback using the Node convention. This is a common pattern, all the following CRUD methods take a callback function like this as the last argument.
```

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
These changes were originally requested by @RandellDawson . Have changed the line so that they can be committed

File: curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-and-save-a-record-of-a-model.english.md (hunk 2 above)
Comment (RandellDawson) [reply curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-and-save-a-record-of-a-model.english.md]: @thecodingaviator Why was the line I suggested (which is the same as yours) not able to be commit by user?  It looks like mine comment was resolved.

Comment (thecodingaviator) [reply curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-and-save-a-record-of-a-model.english.md]: Yes, I marked your comment as resolved. You had actually made a suggestion on the deleted line, i.e. the red line. If you unresolved the comment and try to click on Commit Suggestion, you’ll see what I mean

Comment (thecodingaviator) [reply curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-and-save-a-record-of-a-model.english.md]: Yes, I marked your comment as resolved. You had actually made a suggestion on the deleted line, i.e. the red line. If you unresolved the comment and try to click on Commit Suggestion, you’ll see what I mean

Comment (RandellDawson) [reply curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-and-save-a-record-of-a-model.english.md]: That is odd.

**Comment by moT01** [comment]
Can you make these changes quick @duytrinhvn?

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
@thecodingaviator and @moT01 I have made the changes on behalf of the user.

**Review by moT01 — APPROVED** [review]
Looks good @duytrinhvn - congratulations on your first contribution to freeCodeCamp! 🎉

**Review by thecodingaviator — COMMENTED** [review]
👍🏻

**Pull request merged**